iree/compiler/Dialect/VM/Target/C/CModuleTarget.cpp

```diff
@@ -699,12 +700,20 @@ static LogicalResult canonicalizeModule(
     modulePasses.addPass(mlir::createCanonicalizerPass());
   }
 
+  // C target specific pass
+
+  // Erase exports annotated with 'emitc.exclude'. This makes testing
+  // of partially supported ops easier. For the DCE pass to remove the
+  // referenced function it must be unused and marked private.
```

Comment:
NIT: Maybe the last sentence could be reformulated? Don't have a suggestion though.